Azure/azure-cli
# [App Service] BREAKING CHANGE: `az appservice ase create`: Update the default App Service Environment to V3

**Related command**

az appservice ase create

**Description**

Currently default ASE kind is set to ASEv2. ASEv2 is retiring in August 2024 and creates of ASEv2 will be blocked starting in January. So we need the default to be updated to ASEv3 to prevent errors and ensure customers are getting onto the latest version.

**Testing Guide**

az appservice ase create -g <rg> -n <asename> --vnet-name <vnet> --subnet <subnet> : use this to create an ASEv3 since the default is now set to ASEv3. You no longer need to include the kind parameter for ASEv3 creates.

**History Notes**


[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.
